### PR TITLE
docs: update tests/README.md

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- update Markdown format
- update contents
- add link to the forum thread `Any tutorials on Unit testing in CI4?`

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: admin/module/tests/README.md [hunk 1]
```diff
@@ -1,62 +1,81 @@
...
 
-	> ln -s ./vendor/bin/phpunit ./phpunit
+```console
+> ln -s ./vendor/bin/phpunit ./phpunit
+```
 
-You also need to install [XDebug](https://xdebug.org/index.php) in order
-for code coverage to be calculated successfully.
+You also need to install [XDebug](https://xdebug.org/docs/install) in order
+for code coverage to be calculated successfully. After installing `XDebug`, you must add `xdebug.mode=coverage` in the **php.ini** file to enable code coverage.
```
Comment (samsonasik) [inline admin/module/tests/README.md:32]: in phpunit.xml can be configured as well iirc:

```xml
<env name="XDEBUG_MODE" value="coverage"/>
```

Comment (kenjis) [reply admin/module/tests/README.md:32]: No, it does not work.
The code `<env name="XDEBUG_MODE" value="coverage"/>` is already described in CI4's `phpunit.xml.dist`.

Comment (kenjis) [reply admin/module/tests/README.md:32]: If I remove `xdebug.mode=coverage` in php.ini:
```console
$ vendor/bin/phpunit tests/system/CodeIgniterTest.php 
PHPUnit 9.5.21 #StandWithUkraine

Runtime:       PHP 8.0.21 with Xdebug 3.1.5
Configuration: /Users/kenji/work/codeigniter/CodeIgniter4/phpunit.xml

Code coverage needs to be enabled in php.ini by setting 'xdebug.mode' to 'coverage'
```

But the following works:
```console
$ XDEBUG_MODE=coverage vendor/bin/phpunit tests/system/CodeIgniterTest.php 
PHPUnit 9.5.21 #StandWithUkraine

Runtime:       PHP 8.0.21 with Xdebug 3.1.5
Configuration: /Users/kenji/work/codeigniter/CodeIgniter4/phpunit.xml

........................                                          24 / 24 (100%)

Time: 00:05.303, Memory: 46.00 MB

OK (24 tests, 26 assertions)

Generating code coverage report in Clover XML format ... done [00:10.618]
```

Comment (samsonasik) [reply admin/module/tests/README.md:32]: Ok 👍

**Review by samsonasik — APPROVED** [review]

File: admin/module/tests/README.md [hunk 2]
```diff
@@ -1,62 +1,81 @@
...
+More details on a test database setup are in the
+[Testing Your Database](https://codeigniter4.github.io/userguide/testing/database.html) section of the documentation.
 
 ## Running the tests
 
 The entire test suite can be run by simply typing one command-line command from the main directory.
 
-	> ./phpunit
+```console
+> ./phpunit
```
Comment (MGatner) [inline admin/module/tests/README.md:48]: How does this work on Linux? I don't think the executable ever ends up in the project root.

File: admin/module/tests/README.md [hunk 3]
```diff
@@ -1,62 +1,81 @@
...
+```
 
-You can limit tests to those within a single test directory by specifying the 
-directory name after phpunit. 
+You can limit tests to those within a single test directory by specifying the
+directory name after phpunit.
 
-	> ./phpunit app/Models
+```console
+> ./phpunit app/Models
```
Comment (MGatner) [inline admin/module/tests/README.md:61]: While this is technically fine I don't know if anyone bundling tests into their code directories in PHP. A better example might be:
```suggestion
> ./phpunit tests/Models
```

File: admin/module/tests/README.md (hunk 2 above)
Comment (kenjis) [reply admin/module/tests/README.md:48]: We create a symlink.
See https://github.com/codeigniter4/CodeIgniter4/pull/6274/files/8fe0acad8f8eaf8ffdbd67b176ea818422842397#diff-090405bd51245535c38ef723337f3525aad361cde9778a6f903b71cf9d5f802dR28

**Pull request merged**